client/components/Main/ItemEditor/ItemManager.ts

```diff
@@ -415,7 +415,22 @@ export class ItemManager {
                 const newState = {
                     initialValues: original,
                     diff: diff,
-                    dirty: this.editor.isDirty(initialValues, autosaveItem),
+                    dirty: (() => {
+                        // if associated_plannings in autosaveItem is undefined
+                        // we treat that as there's been no changes to associated_plannings
+                        if (initialValues.type === 'event' && autosaveItem.associated_plannings === undefined) {
+                            return this.editor.isDirty(
+                                initialValues,
+                                autosaveItem,
+                                false,
+                            );
+                        } else {
+                            return this.editor.isDirty(
+                                initialValues,
+                                autosaveItem,
+                            );
+                        }
```

Comment:
Minor improvement/simplification. Thoughts?
```suggestion
                        const ignoreAssociatedPlannings = initialValues.type === 'event' && autosaveItem.associated_plannings === undefined;
                        
                        return this.editor.isDirty(
                            initialValues,
                            autosaveItem,
                            ignoreAssociatedPlannings ? false : undefined,
                        );
```

Reply:
way cleaner